src/web/app/pages-admin/admin-search-page/admin-search-page.component.html

```diff
@@ -94,17 +106,30 @@
       <tbody>
         <ng-container *ngFor="let student of students; let i = index">
           <tr (click)="student.showLinks = !student.showLinks">
-            <td>{{ student.courseId }} [{{ student.section }}] ({{ student.team }})</td> <!-- TODO: tooltip course name -->
+            <td>
+              {{ student.courseId }} [{{ student.section }}] ({{ student.team }})
+              <br/>
```

Comment:
Self-closing HTMLs should not be closed.
```suggestion
              <br>
```